exercises/sum.cpp

```diff
@@ -6,3 +6,30 @@
   Insert the second number: 2
   Sum: 3
 */
+
+#include <iostream> 
+using namespace std; 
+
+int somma (int x, int y){
+	return x+y; 
+}
+
+int Inserire(){
+	int x; 
+	
+	cin>>x; 
+	
+	return x;  
+}
+
+
+int main(){
+	int a, b; 
+	
+	cout<<"inserisci due numeri: "; 
+	a=Inserire(); 
+	b=Inserire(); 
+	
+	cout<<"la somma tra due numeri e'': " <<somma(a,b); 
+	
```

Comment:
manca `return 0`

Reply:
potresti sostituire 0 con EXIT_SUCCESS in modo da migliorare la leggibilità del codice
```c++
#define EXIT_SUCCESS 0
...
return EXIT_SUCCESS;
```

il resto mi sembra ok, ottimo :+1: